src/workerd/io/compatibility-date.capnp

```diff
@@ -377,4 +377,10 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
       $compatDisableFlag("no_nodejs_als");
   # Enables the availability of the Node.js AsyncLocalStorage API independently of the full
   # node.js compatibility option.
+
+  queuesJsonMessages @42 :Bool
+      $compatEnableFlag("queues_json_messages")
+      $compatDisableFlag("no_queues_json_messages")
+      $compatEnableDate("2024-03-04");
```

Comment:
This date is too soon: This will not be rolled out to production, much less to the validator, by Monday. Please push this back by two weeks.

If any release has already been built with this compat flag, that release will need to be patched to fix this, otherwise when the validator is updated (presumably after March 4) it will retroactively change the meaning of past compat dates.

Reply:
https://github.com/cloudflare/workerd/pull/1744